src/importexport/internal/flacwriter.cpp

```diff
@@ -23,15 +23,14 @@
 
 using namespace mu::importexport;
 using namespace mu::framework;
-using namespace Ms;
 
-mu::Ret FlacWriter::write(const Score& score, IODevice& destinationDevice, const Options& options)
+mu::Ret FlacWriter::write(const notation::INotation* notation, IODevice& destinationDevice, const Options& options)
```

Comment:
I think, better to use INotationPtr instead of raw pointer

Reply:
> I think, better to use INotationPtr instead of raw pointer

Yes, but it's inconvenient here
https://github.com/musescore/MuseScore/pull/7166/files#diff-bc2452ffd86e64620f2974b5e7fa0a8c1d3bdf00b5adee80b1dec460877c92b0R539

Reply:
I will fix it

Reply:
fixed